ballerina/client.bal

```diff
@@ -1,15 +1,184 @@
+    # + payload - The `Content` to tokenize
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response 
+    remote isolated function generativelanguage\.models\.countTokens(string model, CountTokensRequest payload, map<string|string[]> headers = {}, *GenerativelanguageModelsCounttokensQueries queries) returns http:Response|error {
+        string resourcePath = string `/v1/${getEncodedUri(model)}:countTokens`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, headers);
+    }
+
+    # Generates a text embedding vector from the input `Content` using the specified [Gemini Embedding model](https://ai.google.dev/gemini-api/docs/models/gemini#text-embedding).
+    #
+    # + model - Required. The model's resource name. This serves as an ID for the Model to use. This name should match a model name returned by the `ListModels` method. Format: `models/{model}`
+    # + payload - The `Content` to embed
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response 
+    remote isolated function generativelanguage\.models\.embedContent(string model, EmbedContentRequest payload, map<string|string[]> headers = {}, *GenerativelanguageModelsEmbedcontentQueries queries) returns http:Response|error {
+        string resourcePath = string `/v1/${getEncodedUri(model)}:embedContent`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, headers);
+    }
+
+    # Generates a model response given an input `GenerateContentRequest`. Refer to the [text generation guide](https://ai.google.dev/gemini-api/docs/text-generation) for detailed usage information. Input capabilities differ between models, including tuned models. Refer to the [model guide](https://ai.google.dev/gemini-api/docs/models/gemini) and [tuning guide](https://ai.google.dev/gemini-api/docs/model-tuning) for details.
+    #
+    # + model - Required. The name of the `Model` to use for generating the completion. Format: `models/{model}`.
+    # + payload - The `GenerateContentRequest` to generate content
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response 
+    remote isolated function generativelanguage\.models\.generateContent(string model, GenerateContentRequest payload, map<string|string[]> headers = {}, *GenerativelanguageModelsGeneratecontentQueries queries) returns http:Response|error {
```

Comment:
Seems like the paths are separated by dots rather than slashes, Might need to look into it. Any idea @TharmiganK @lnash94